go.mod

```diff
@@ -0,0 +1,23 @@
+module github.com/kube-queue/job-extension
+
+go 1.15
+
+require (
+	github.com/kube-queue/api v0.0.0-20210623033849-bffe1acb5aa9
+	github.com/sirupsen/logrus v1.6.0 // indirect
+	k8s.io/api v0.18.5
```

Comment:
change to 1.22?

Reply:
yes,  this version can support on k8s 1.22+